src/atc/functions.py

```diff
@@ -2,10 +2,47 @@
 A collection of useful pyspark snippets.
 """
 
+from atc.spark import Spark
 from pyspark.sql.functions import udf
 from pyspark.sql.types import StringType
 import uuid as _uuid
+import pyspark.sql.functions as F
+from pathlib import Path
 
 # Pyspark uuid function implemented as recommended here
 # https://stackoverflow.com/questions/49785108/spark-streaming-with-python-how-to-add-a-uuid-column/50095755
 uuid = udf(lambda: str(_uuid.uuid4()), StringType()).asNondeterministic()  # noqa
+
+def init_dbutils(spark):
```

Comment:
since we use Spark.get() in drop_table_cascade, is it not ok  to use it here also and remove the argument?

Reply:
Like this?

def init_dbutils():
    try:
        from pyspark.dbutils import DBUtils
        dbutils = DBUtils(Spark.get())
    except ImportError:
        import IPython
        dbutils = IPython.get_ipython().user_ns["dbutils"]
    return dbutils